### ISSUE TITLE
role="status" not read out when appended to body

### ISSUE THREAD

**Issue by cordeliadillon** [nvaccess/nvda#5803]
I'm running into an issue with NVDA not reading out status messages that are appended to the body in Firefox. It'll read out text that is added to an existing DOM node with `role="status"`, but won't read out if I create a new node with `role="status"` unless I've specifically added a timeout. This is inconsistent with the behavior of other common browser / screenreader combinations (JAWS+IE, VO+Safari), which read out the element when it's added to the DOM, even if it doesn't have a container. I've created this test page with a few examples here: http://codepen.io/anon/pen/wGGEOM

**Comment by jcsteh** [comment]
The problem with reading live regions (which includes role="status") 
when they're added is that this would cause all regions to be read when 
a page loads or when a major section of the page is added. That could be 
a *huge* flood of info. Imagine a chat site, for example, where the page 
loads populated with chat history. Once upon a time, we could have 
ignored all regions created while the page was "loading", but the 
concept of loading doesn't really exist any more on the modern web; we 
want content to render as soon as it's ready.

**Comment by leonardder** [comment]
I think @jcsteh's https://github.com/nvaccess/nvda/issues/5803#issuecomment-194604503 points in the direction of a won't fix. @cordeliadillon: do you still consider this request relevant?

**Comment by Adriani90** [comment]
this is still reproducible in Firefox 65.0.1 and Chrome 71 with NVDA 2018.4.1.

**Comment by Adriani90** [comment]
I seems we didn't get any further arguments on this discussion. I am closing this issue since we don't have any response from the initial author since three years.
Please comment on this issue and we can reopen it if needed. Thanks.

**Comment by SandraZoocha** [comment]
Hello, I'm having the same issue (I guess). In my case, I've a view with some filters, and I need NV Access to read the number of results after filtering. The page is adding a <p class="sr-only" role="status">2 results</p>.
If I test this with VoiceOver I can hear how it says 'New page load' '2 results'. But testing it with NV Access it doesn't read anything. 
Any help here?
